### PR TITLE
Code review dashboard: Filter by a card name

### PR DESCRIPTION
**Pull request by Ngoakor12**
Related issues: [[Add another filter into the left hand panel of the review Q page - this will be for the name of the card](https://www.notion.so/Add-another-filter-into-the-left-hand-panel-of-the-review-Q-page-this-will-be-for-the-name-of-the--22aadbf1af3c47808764e9c068123f4f)]

## Description:

What are you up to? Fill us in :)

## Screenshots/videos
image

image

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by FaithMo — APPROVED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

File: frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx [hunk 1]
```diff
@@ -119,6 +128,25 @@ export default function Presentation({
...
+        <Paper>
+          <TextField
+            className={classes.filterByNameTextField}
+            variant="outlined"
+            placeholder="Card name"
+            value={cardNameFilterSearchTerm}
+            onChange={handleOnchangeCardNameFilter}
+            fullWidth
+          />
+          <FilterByNames
```
Comment (sheenarbw) [inline frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx]: Just a textbox is sufficient here.  It should work similarly to how the textboxes here work: https://tilde-front-dot-umuzi-prod.nw.r.appspot.com/users 

Or do you think that being able to toggle a bunch of things will be useful? It seems like a very heavy solution to me

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This seems a bit heavy. I think I should have been more clear in the issue though - Surely a textbox would be enough here?

File: frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx (hunk 1 above)
Comment (Ngoakor12) [reply frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx]: No, just a misunderstanding. I've updated it to only use a textbox.

**Review by FaithMo — CHANGES_REQUESTED** [review]
the storybook says " `TypeError: undefined is not a function` Is this issue going to be resolved by the other PR or it needs to be addressed here ?

**Comment by Ngoakor12** [comment]
> the storybook says " `TypeError: undefined is not a function` Is this issue going to be resolved by the other PR or it needs to be addressed here ?

Forgot to mention it. Yes, it will be addressed by the [this PR](https://github.com/Umuzi-org/Tilde/pull/636) which is closer to being merged.

**Review by FaithMo — APPROVED** [review]

**Review by FaithMo — APPROVED** [review]
merge conflicts resolved

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**